components/printful/sources/order-status-updated-instant/order-status-updated-instant.mjs

```diff
@@ -0,0 +1,27 @@
+import common from "../common/base.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  ...common,
+  key: "printful-order-status-updated-instant",
+  name: "New Order Status Updated (Instant)",
```

Comment:
```suggestion
  name: "Order Status Updated (Instant)",
```
Starting with 'new' is a guideline, not a rule - I think for 'updated' items such as in this case, as well as 'deleted', 'moved' etc. the 'new' can be confusing and should be avoided